documents/fund.md

```diff
@@ -0,0 +1,87 @@
+## Funds
+
+```rb
+require "razorpay"
+Razorpay.setup('key_id', 'key_secret')
+```
+
+### Create a fund account
+```rb
+data = {
```

Comment:
variable name need to be consistent. please `change` data to `para_attr`

Reply:
done